lib/plugins/pluginbase.js

```diff
@@ -50,6 +50,30 @@ function updatePillText(updatedText, label, info, major) {
   }
 }
 
+function roundInsulinForDisplayFormat(iob) {
```

Comment:
since this is pump specific we should make it an option, been thinking of how we can pass options to the plugins.  Maybe something like `IOB_ROUNDING="medtronic"`?